src/DynamoCoreWpf/Views/Core/NodeView.xaml

```diff
@@ -17,7 +18,8 @@
 
     <Grid Name="grid"
           x:FieldModifier="public"
-          HorizontalAlignment="Left">
+          HorizontalAlignment="Left"
+          Visibility="{Binding IsCollapsed, Converter={StaticResource InverseBoolToVisibilityCollapsedConverter}}">
```

Comment:
Prolly dont need it, but `UpdateSourceTrigger=PropertyChanged`? Any thoughts on why sometimes that is needed and why sometimes things just update without it?

Reply:
Depends on which property you are dealing with, most will by default have `UpdateSourceTrigger` set to `PropertyChanged`, but some like `Text` will have it to `LostFocus` or something else.

here we dont need it as `Visibility` is already set to `PropertyChanged`